source/tests/system/nirfmxinstr_driver_api_tests.cpp

```diff
@@ -165,6 +165,28 @@ TEST_F(NiRFmxInstrDriverApiTests, SetAndGetFrequencyReferenceSource_ReturnsFrequ
   EXPECT_EQ(response.attr_val(), "OnboardClock");
 }
 
+TEST_F(NiRFmxInstrDriverApiTests, GetModelName_ReturnsModelName)
+{
+  const auto session = init_session(stub(), PXI_5663E);
+  const auto response = client::get_attribute_string(stub(), session, "", NiRFmxInstrAttribute::NIRFMXINSTR_ATTRIBUTE_INSTRUMENT_MODEL);
+
+  EXPECT_SUCCESS(session, response);
+  EXPECT_EQ(response.attr_val(), "NI PXIe-5663E");
+}
+
+TEST_F(NiRFmxInstrDriverApiTests, TimestampFromValuesRoundTrip_SucceedsWithOriginalValues)
+{
+  constexpr auto SECONDS_SINCE_1970 = 10000;
```

Comment:
Super nit-picky but maybe just name this UNIX_TIMESTAMP, since technically this low of a UNIX code is still Jan 1st 1970

Reply:
Hah, I guess that's more concise than `SECONDS_SINCE_MIDNIGHT_JANUARY_FIRST_1970`